packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -78,4 +80,39 @@ export default class BigQuery extends SqlIntegration {
   castUserDateCol(column: string): string {
     return `CAST(${column} as DATETIME)`;
   }
+
+  async getInformationSchema(datasourceType: string, projectId: string) {
```

Comment:
Some feedback with this method:

- let's use an explicit return type declaration for readability, e.g. `async getInformationSchema(datasourceType: string, projectId: string): { /* some type */ }[]`. It's not clear what is returned from this function.
- considering many data sources will have this, do we want to add this to the abstract class `SqlIntegration` as `async getInformationSchema(datasourceType: string, projectId: string): unknown`? This may make it easier in the future for someone who needs to add a new data source to fill out all the override methods form the abstract class.